pysqldb3/tests/test_query_creates_table.py

```diff
@@ -3,97 +3,141 @@
 
 
 class TestQueryCreatesTablesSql():
+    def test_complicated(self):
+        query_string = """
+            -- create table test.test; 
+            /* 
+            CREATE TABLE WTF.TEST as select ; 
+            */
+            --/* 
+            CREATE TABLE Schema.fake_out as select; 
+            --*/
+            create table server.db.schema.table as select ;
+            create table [server].db.schema.[table] as select ;
+            create table server.[db].schema.[CapitalTable] as select ;
+            create table server.[db].schema.BadCapitalTable as select ;
+            create table server.db."schema".[Capital Table] as select ;
+            create table server.db.schema.[1-able] (id_ int) ;
+            create table [server]."db".schema._table as select ;
+            create table server.db.schema.[-_ta^b l*e] as select ;
+            create table db.schema."-_ta&$b l*e" as select ;
+
+            create table [schema]."123456-_ta&$b l*e" as select ;
+            create table [123456-_ta&$b l*e] as select ;
+                     
+                """
```

Comment:
as an extra test for robustness, it would be good to have at least 1 `--` comment that shares a line with code, like:
 `create table server.db.schema.table as select; -- this line creates a table`

Reply:
@cindyatnycdot just pushed an update, test passing